components/look-at/index.js

```diff
@@ -39,6 +39,7 @@ AFRAME.registerComponent('look-at', {
   init: function () {
     this.target3D = null;
     this.vector = new THREE.Vector3();
+    this.isCamera = this.el.getObject3D('camera') ? true : false;
```

Comment:
whats the best way to tell if camera is initialized?   What about in update?   We really only need to check it once.  Don't know how expensive checking in tick would be?